core/src/main/java/org/infinispan/remoting/inboundhandler/GlobalInboundInvocationHandler.java

```diff
@@ -67,7 +68,9 @@ private static ExceptionResponse exceptionHandlingCommand(Throwable throwable) {
    public void handleFromCluster(Address origin, ReplicableCommand command, Reply reply, DeliverOrder order) {
       command.setOrigin(origin);
       try {
-         if (command instanceof CacheRpcCommand) {
+         if (command.getCommandId() == HeartBeatCommand.COMMAND_ID) {
+            reply.reply(null);
```

Comment:
I don't like this reply null, seems like an error when reading, but I don't know if I have any other suggestion or alternative to offer

Reply:
it is ok. it is an empty reply. the `DefaultResponseGenerator` does the same trick to avoid serializing a Response :)